dafni.toml

```
@@ -0,0 +1,32 @@
+[autokube]
+disableBackendsPrefix=true
+
+[backends]
+VISUALISATION_API_URL="https://dafni-nivs-nivs-api-review-dev-hkw12f.staging.dafni.rl.ac.uk"
+LOGIN_URL="https://dafni-dss-login-app-jwt-review-dev-h79t6w.staging.dafni.rl.ac.uk"
+DISCOVERY_API_URL="https://dafni-search-and-discovery-api-review-dev-4jxwt5.staging.dafni.rl.ac.uk"
+DSS_AUTH="https://dafni-dss-dssauth-review-dev-o2yn5p.staging.dafni.rl.ac.uk"
+NODE_ENV="dev"
+
+[[configset]]
+environment="staging"
+
+[configset.backends]
+VISUALISATION_API_URL="https://dafni-nivs-nivs-api.staging.dafni.rl.ac.uk"
+LOGIN_URL="https://login.staging.dafni.rl.ac.uk"
+DISCOVERY_API_URL="https://dafni-search-and-discovery-api.staging.dafni.rl.ac.uk"
+DSS_AUTH="https://dafni-dss-dssauth.staging.dafni.rl.ac.uk"
+NODE_ENV="production"
```

I find it helps to indent the configuration within an environment just so its easier to see the separation so like:

```suggestion
[[configset]]
environment="staging"

  [configset.backends]
  VISUALISATION_API_URL="https://dafni-nivs-nivs-api.staging.dafni.rl.ac.uk"
  LOGIN_URL="https://login.staging.dafni.rl.ac.uk"
  DISCOVERY_API_URL="https://dafni-search-and-discovery-api.staging.dafni.rl.ac.uk"
  DSS_AUTH="https://dafni-dss-dssauth.staging.dafni.rl.ac.uk"
  NODE_ENV="production"
```

---

(and similar for production below)